go.mod

```diff
@@ -1,17 +1,19 @@
 module github.com/NVIDIA/k8s-operator-libs
 
-go 1.20
+go 1.21
```

Comment:
There's an article about this change here: https://go.dev/blog/toolchain

Reply:
just to make sure i understand what these two directives mean

1. consumers of this module should have go 1.21 or newer
2. when working on this module go 1.21.5 will be used


is that correct ?

Reply:
Yeah - that's the summary. To be honest though I don't think we need this here and it's non-compulsory. I'm going to take it out of this PR. If someone wants to add it it might be better to do it seperately.